parts/k8s/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -169,6 +169,7 @@ apt_get_install() {
     retries=$1; wait_sleep=$2; timeout=$3; shift && shift && shift
     for i in $(seq 1 $retries); do
         wait_for_apt_locks
+        export DEBIAN_FRONTEND=noninteractive
```

Comment:
This is to skip the prompt that says "I noticed you already have a configuration file for this package, should I use it? or overwrite it?" etc. Because we are pre-loading the configuration before apt-installing chrony.

Reply:
:+1: This should always be set IMHO before `apt` commands in scripts and Dockerfiles.